nyan/nyan_tool.cpp

```diff
@@ -189,30 +205,29 @@ int run(flags_t flags, params_t params) {
 
 void help() {
 	std::cout << "\x1b[32;1mnyan\x1b[m - "
-	             "\x1b[32;1my\x1b[met "
-	             "\x1b[32;1ma\x1b[mnother "
-	             "\x1b[32;1mn\x1b[motation "
-	             "-- tool" << std::endl
-	          << std::endl
-	          << "usage:" << std::endl
-	          << "-h --help                  -- show this" << std::endl
-	          << "-f --file <filename>       -- file to load" << std::endl
-	          << "-b --break                 -- debug-break on error" << std::endl
-	          << "   --test-parser           -- test the parser" << std::endl
-	          << "   --echo                  -- print the ast" << std::endl
-	          << "" << std::endl;
+				 "\x1b[32;1my\x1b[met "
+				 "\x1b[32;1ma\x1b[mnother "
+				 "\x1b[32;1mn\x1b[motation "
+				 "-- tool"
+			  << std::endl
+			  << std::endl
+			  << "usage:" << std::endl
+			  << "-h --help                  -- show this" << std::endl
+			  << "-f --file <filename>       -- file to load" << std::endl
+			  << "-b --break                 -- debug-break on error" << std::endl
+			  << "   --test-parser           -- test the parser" << std::endl
+			  << "   --echo                  -- print the ast" << std::endl
+			  << "" << std::endl;
```

Comment:
can we please not do lots of reindentation in this small bugfix :)

Reply:
oh no the autoformatter strikes again!

Reply:
fixed

Reply:
I did the format changes in #120